demo/src/lib/stackblitz/angular/tsconfig.json

```diff
@@ -8,7 +8,6 @@
 		"skipLibCheck": true,
 		"esModuleInterop": true,
 		"resolveJsonModule": true,
-		"moduleResolution": "node",
```

Comment:
This change breaks angular samples in stackblitz.

Reply:
Thanks for noticing !